package.json

```diff
@@ -14,7 +14,7 @@
     ]
   },
   "scripts": {
-    "reset": " git clean -Xfd; git clean -fd; pnpm i && pnpm build",
+    "reset": "git clean -Xfd; git clean -fd; pnpm i && pnpm build > /dev/null 2>&1; pnpm i -f; pnpm build; pnpm sync",
```

Comment:
@NullVoxPopuli had to reinstate your original reset script (main is broken without this fix too)

Reply:
Hmm sus. I'll investigate